apps/backend/src/mcp/resources/proxy_configs.py

```
@@ -0,0 +1,634 @@
+        elif device and path == 'summary':
+            return await _get_proxy_summary_resource(device)
+        
+        else:
+            raise ValueError(f"Unknown SWAG URI format: {uri}. Use swag://service_name or swag://device/directory")
+        
+    except Exception as e:
+        logger.error(f"Error getting proxy config resource {uri}: {e}")
+        return {
+            'error': str(e),
+            'uri': uri,
+            'timestamp': datetime.now(timezone.utc).isoformat(),
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)